x-pack/legacy/plugins/siem/public/components/page/hosts/hosts_table/index.tsx

```diff
@@ -90,47 +89,63 @@ const rowItems: ItemsPerRow[] = [
     numberOfRow: 10,
   },
 ];
+const getSorting = (
```

Comment:
After some research and experiment, I think that we can do the memoization with the hooks `useMemo` like it was before with `memoizeOne` like that. Tell me what do you think?
```
const hostsColumns = useMemo(() => getMemoizeHostsColumns(type, indexPattern), [
      type,
      indexPattern,
    ]);

    const sorting = useMemo(() => getSorting(`${sortField}-${direction}`, sortField, direction), [
      sortField,
      direction,
    ]);

    return (
      <PaginatedTable
        activePage={activePage}
        columns={hostsColumns}
        dataTestSubj={`table-${tableType}`}
        headerCount={totalCount}
        headerTitle={i18n.HOSTS}
        headerUnit={i18n.UNIT(totalCount)}
        id={id}
        isInspect={isInspect}
        itemsPerRow={rowItems}
        limit={limit}
        loading={loading}
        loadPage={newActivePage => loadPage(newActivePage)}
        onChange={onChange}
        pageOfItems={data}
        showMorePagesIndicator={showMorePagesIndicator}
        sorting={sorting}
        totalCount={fakeTotalCount}
        updateLimitPagination={newLimit =>
          updateTableLimit({
            hostsType: type,
            limit: newLimit,
            tableType,
          })
        }
        updateActivePage={newPage =>
          updateTableActivePage({
            activePage: newPage,
            hostsType: type,
            tableType,
          })
        }
      />
    );
```

Reply:
Man, i didn't see that hook all the way down there in the docs. good call. ill make this change 👍

Reply:
Also, I'm going to remove `getMemoizeHostsColumns` from this file and just call `getHostsColumns` directly.